modules/svneverever/__main__.py

```diff
@@ -255,12 +256,17 @@ def command_line():
 
     return args
 
+def login(realm, username, may_save):
+    username = six.input("Please enter your username: ")
+    password = getpass.getpass()
+    return True, username, password, False
 
 def main():
     args = command_line()
 
     # Build tree from repo
     client = pysvn.Client()
+    client.callback_get_login = login
```

Comment:
It would be nice to guard this line by a new command line switch, either `--interactive` or `--non-interactive`.